templates/repo/settings/branches.tmpl

```diff
@@ -37,19 +37,24 @@
 			</h4>
 
 			<div class="ui attached segment">
-				<div class="flex-list">
+				<div class="flex-list" id="protected-branches-list" data-priority-url="{{$.Repository.Link}}/settings/branches/priority">
```

Comment:
Maybe `data-priority-url` should be `data-update-priority-url

Reply:
4b9f09c1d0e7841ea61d18a615c8ae6f2a3adcdb